bin/alluxio

```diff
@@ -257,13 +257,8 @@ function main {
     runJavaClass "$@"
   ;;
   "runTest")
+    PARAMETER="configured" # instructs runTests to use the configured master address
```

Comment:
I think it is okay, `TestRunner` is not a public api, and we should never be running cross version tests.